android/java/org/chromium/chrome/browser/webcompat_reporter/WebcompatReporterServiceFactory.java

```diff
@@ -34,8 +34,8 @@ public static WebcompatReporterServiceFactory getInstance() {
     private WebcompatReporterServiceFactory() {}
 
     public WebcompatReporterHandler getWebcompatReporterHandler(
-            ConnectionErrorHandler connectionErrorHandler) {
-        Profile profile = Utils.getProfile(false); // Always use regular profile
+            ConnectionErrorHandler connectionErrorHandler, boolean isIncognito) {
+        Profile profile = Utils.getProfile(isIncognito);
```

Comment:
Do we want to use5rs to report private tab details ?

Reply:
yes users can send webcompat report from the incognito mode, but in that case we should not save contact info in the preferences